src/transport/raw/Base.h

```diff
@@ -33,6 +33,14 @@
 namespace chip {
 namespace Transport {
 
+class RawTransportDelegate
+{
+public:
+    virtual ~RawTransportDelegate() {}
+    virtual void HandleMessageReceived(const PacketHeader & packetHeader, const Transport::PeerAddress & peerAddress,
+                                       System::PacketBufferHandle && msg) = 0;
```

Comment:
This should probably also take the handle by value

Reply:
Done